CRM/Contribute/BAO/Contribution.php

```diff
@@ -5297,8 +5297,9 @@ public static function getContributionTokenValues($id, $messageToken) {
     $result = civicrm_api3('Contribution', 'get', ['id' => $id]);
     // lab.c.o mail#46 - show labels, not values, for custom fields with option values.
     if (!empty($messageToken)) {
+      $messageTokenContribution = CRM_Utils_Array::value('contribution', $messageToken);
```

Comment:
```suggestion
      $messageTokenContribution = CRM_Utils_Array::value('contribution', $messageToken, []);
```
given we are using an array function we probably should return an array by default rather than NULL

Reply:
Thanks, that's a very good point. Seems like a was a little too quick to respond :)

Reply:
@bjendres Also we should use the coalesce operator instead of the `CRM_Utils_Array::value()` function.

Eg. 
```
$messageTokenContribution = $messageToken['contribution'] ?? [];
```